packages/xrpl/src/client/RequestManager.ts

```diff
@@ -93,10 +99,16 @@ export default class RequestManager {
       newId = request.id
     }
     const newRequest = JSON.stringify({ ...request, id: newId })
-    const timer = setTimeout(
-      () => this.reject(newId, new TimeoutError()),
-      timeout,
-    )
+    // Typing required for Jest running in browser
```

Comment:
Out of curiosity, do you remember what happens if the type isn't specified? (Is the issue that typescript can't compile it effectively without the type, and so it uses a forbidden function?) 

(If not it's not worth re-creating the issue)

Reply:
`setTimeout` has a different function signature in nodejs versus the browser. If you type it explicitly as `Node.Timeout` it is incorrect.